listener.c

```diff
@@ -243,7 +242,7 @@ evconnlistener_new_bind(struct event_base *base, evconnlistener_cb cb,
 	}
 #endif
 	if (support_keepalive) {
-		if (setsockopt(fd, SOL_SOCKET, SO_KEEPALIVE, (void*)&on, sizeof(on))<0)
+		if (evutil_set_tcp_keepalive(fd, 1, 300) < 0)
```

Comment:
Ok for now, but it will be useful to make this timeout configurable

Reply:
Added a TODO for it.